examples/video/instream/videoModule/videojs/video-module-videojs.html

```diff
@@ -0,0 +1,280 @@
+                        url: 'https://prebid.adnxs.com/pbc/v1/cache'
+                    },
+                    targetingControls: {
+                        allowTargetingKeys: ['BIDDER', 'AD_ID', 'PRICE_BUCKET', 'SIZE', 'DEAL', 'SOURCE', 'FORMAT', 'UUID', 'CACHE_ID', 'CACHE_HOST', 'ADOMAIN']
+                    },
+                });
+
+                pbjs.addAdUnits(adUnits);
+
+                pbjs.onEvent('videoSetupComplete', e => {
+                    // Load media with its Metadata when the video player is done instantiating.
+                    videojs('player').loadMedia({
+                        id: 'XYXYXYXY',
+                        src: 'http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/SubaruOutbackOnStreetAndDirt.mp4',
+                        title: 'Subaru Outback On Street And Dirt',
+                        description: 'Smoking Tire takes the all-new Subaru Outback to the highest point we can find in hopes our customer-appreciation Balloon Launch will get some free T-shirts into the hands of our viewers.',
+                        type: 'video/mp4'
+                    });
+                });
+
+                pbjs.onEvent('videoSetupFailed', e => {
+                    console.log('The player setup failed: ', e);
+                });
+
+                pbjs.onEvent('videoAdRequest', (e) => {
+                    console.log('An ad request was made: ', e);
+                });
+
+                pbjs.onEvent('videoBidError', e => {
+                    console.log('An Ad Error came from a Bid: ', e);
+                });
+
+                pbjs.onEvent('videoBidImpression', e => {
+                    console.log('An Ad Impression came from a Bid: ', e);
+                });
+
+                pbjs.requestBids(adUnits);
+            });
+        &lt;/script&gt;
+
+                </pre>
+              </div>
+
+              <h4>Place this code in the page body.</h4>
+              <div class="pb-code-hl-wrap">
+                <!--body code example-->
+                <pre class="pb-code-hl" >
+                  <!--put body html and javascript here-->
+                  &lt;div class="example-video-container"&gt;
+                    &lt;video id="player" class="vjs-big-play-centered"&gt;&lt;/video&gt;
+                  &lt;/div&gt;
+                </pre>
+              </div>
+            `;
+        </script>
+    </div>
+</div>
+
+{% if jekyll.environment == "production" %}
+<script>
+    window.onetrustLoaded.then(() => {
+        OneTrust.InsertHtml(exampleHTML, 'videoExample', null, {deleteSelectorContent: true}, 'C0003');
+    });
+</script>
+{% else %}
+<script>
+    const videoExampleDiv = document.getElementById('videoExample');
+    videoExampleDiv.innerHTML = exampleHTML;
+</script>
+{% endif %}
+
+<!--video player code-->
+<script>
+    // Setup ad units
+    var adUnits = [{
+        code: 'div-gpt-ad-51545-0',
+        mediaTypes: {
+            video: {}
```

Comment:
Without a "context" specified, some bidders won't bid properly. For example appnexus. This should be `context: 'instream'`

Reply:
not required for video module since the video module will fill this out on behalf of the publisher. 
https://github.com/prebid/Prebid.js/blob/master/modules/videoModule/index.js#L136